### DISCUSSION TITLE
Clarification on the Role of sa in Modelica.Mechanics.Rotational.Interfaces.PartialFriction

### DISCUSSION THREAD

**Discussion by awecefil** [modelica/ModelicaStandardLibrary#4509] [Q&A]
Hello,

I am working with the `Modelica.Mechanics.Rotational.Interfaces.PartialFriction` model from the Modelica Standard Library and have encountered the variable `sa`, described as the "path parameter of friction characteristic." The documentation mentions that `sa` is involved in calculating the relative angular acceleration (a_relfric) and appears to play a role in state transitions (e.g., from stuck to sliding). However, I don't know how `sa` is typically determined.

Here are my specific questions:

1. What does `sa` conceptually represent in this context, and how is it expected to be computed or provided in practical applications?
2. Are there example models or scenarios in the Standard Library where sa is explicitly defined or used, which could help clarify its role?

I appreciate your help in understanding this parameter and its typical usage within friction models.

Thank you!

**Comment by tobolar** [comment]
Did you read `Modelica.Mechanics.Rotational.UsersGuide.ModelingOfFriction`?

**Reply by awecefil** [reply]
Yes, I have read `Modelica.Mechanics.Rotational.UsersGuide.ModelingOfFriction` and it seems that `sa` is a variable which decides startForward or startBackward is true. But I don't understand where `sa` is been defined (how to calculate its value)? and what it means in this friction model? ex: v represents velocity and its unit is m/s.

**Reply by tobolar** [reply]
@awecefil Please check also 'Friction and loss elements' in paper 'Package PowerTrain: A Modelica library for modeling and simulation of vehicle power trains' (https://modelica.org/events/workshop2000/proceedings/Otter.pdf)
